cellcom/src/backend/status.rs

```diff
@@ -0,0 +1,244 @@
+use std::{sync::OnceLock, time::Duration};
+
+use eyre::Result;
+use orb_endpoints::{v2::Endpoints as EndpointsV2, Backend, OrbId};
+use orb_security_utils::reqwest::reqwest::blocking::Client;
+use serde::{Deserialize, Serialize};
+use tracing::debug;
+
+use crate::data::{CellularInfo, WifiNetwork};
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct OrbStatusV2 {
+    pub orb_id: Option<String>,
+    pub location_data: Option<LocationDataV2>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct LocationDataV2 {
+    pub wifi: Option<Vec<WifiDataV2>>,
+    pub cell: Option<Vec<CellDataV2>>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct WifiDataV2 {
+    pub ssid: Option<String>,
+    pub bssid: Option<String>,
+    pub signal_strength: Option<i32>,
+    pub channel: Option<u32>,
+    pub signal_to_noise_ratio: Option<i32>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct CellDataV2 {
+    pub mcc: Option<u32>,
+    pub mnc: Option<u32>,
+    pub lac: Option<String>,
+    pub cell_id: Option<u32>,
+}
+
+pub fn client() -> &'static Client {
+    static CLIENT: OnceLock<Client> = OnceLock::new();
+    CLIENT.get_or_init(|| {
+        let builder = orb_security_utils::reqwest::blocking::http_client_builder()
+            .timeout(Duration::from_secs(60))
+            .user_agent("orb-cellcom");
+        builder.build().expect("Failed to build client")
+    })
+}
+
+pub fn get_location(
+    orb_id: &OrbId,
+    cellular_info: &CellularInfo,
+    wifi_networks: &[WifiNetwork],
+) -> Result<()> {
+    let request = build_status_request(orb_id, cellular_info, wifi_networks)?;
+
+    debug!(
+        request = ?serde_json::to_string(&request)?,
+        "Sending status v2 request"
+    );
+
+    let backend = Backend::from_env_or_build_type::<{ cfg!(feature = "stage") }>();
```

Comment:
I am gonna remove this function, it was a mistake. Doesn't make sense anymore after the analysis backend was added. Can you use `ORB_BACKEND` or explicitly accept the backend enum as an arg?